src/ert/resources/workflows/jobs/internal-gui/scripts/gen_data_rft_export.py

```diff
@@ -284,7 +249,7 @@ def getArguments(self, parent, storage):
                     output_path_model.getPath(),
                     trajectory_model.getPath(),
                     ensemble_list,
-                    infer_iteration_check.isChecked(),
+                    True,
```

Comment:
Think you want to remove this from the list now.

Reply:
Done, thanks.
`drop_const_cols` was read from `workflow_args` which a string, so it would be truthy even when "False".
I modified the test so that it checks the drop constant columns checkbox and produces the same results as before.